pom.xml

```diff
@@ -30,6 +30,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
```

Comment:
what does this module do?

Reply:
I'm not really sure that one, I believed it was to start any of the dependencies I use but I realized that there are other dependencies that already start services like JPA or Web. Maybe this one is not necesary, I'll fix it